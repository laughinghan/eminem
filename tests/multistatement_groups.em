let .refrain ^saying (println saying; println saying)
refrain "Two by two, hands of blue"

(
  let .a 1
  let .b 2
  let .c 3
)
print "a: " a ", b: " b ", c: " c ; println ""

println (println "first"; "second")

print "empty group is " () ; println ""

print "split across " \ # continue onto the next line
"two lines
"
