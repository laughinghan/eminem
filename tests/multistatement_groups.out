Two by two, hands of blue
Two by two, hands of blue
a: 1, b: 2, c: 3
first
second
empty group is undefined
split across two lines
